.gitignore

```diff
@@ -59,7 +59,7 @@ BenchmarkDotNet.Artifacts/
 # .NET Core
 project.lock.json
 project.fragment.lock.json
-artifacts/
+#artifacts/

```

Comment:
We can maybe find a better way to do this (ignore all `artifacts/` except for the DMX ones? Or vice versa

Reply:
This should be ok for now